test/test_torch.py

```diff
@@ -4948,6 +4948,15 @@ def _test_cat_empty(self, use_cuda=False):
     def test_cat_empty(self):
         self._test_cat_empty(self)
 
+    def test_cat_big(self):
```

Comment:
Should this test be marked as slow or something like that? Also, how much memory does it need, > 2GB?

Reply:
The test needs 5 GB and takes about 8 seconds on my machine. Do you know if there is a good way to mark tests as not running in memory-constrained environments?

Alternatively we can just land this without the test. What do you think?

Reply:
I think we should have the test. Maybe at some point in the future we might consider adding a flag for memory intensive tests, both on the cpu and you